vmcnet/models/equivariance.py

```diff
@@ -169,10 +171,16 @@ class FermiNetOneElectronLayer(Module):
             systems) or should be a Sequence with length 1 whose element is less than
             the total number of electrons.
         ndense (int): number of dense nodes
+        num_heads (int): number of heads. If num_heads == 1, then the multi-head
```

Comment:
Same here as above - can we make this an optional argument after use_transformer?

Reply:
sgtm